tests/dragonfly/connection_test.py

```diff
@@ -64,6 +64,7 @@ async def stop(self, timeout=0.1):
 @pytest.mark.asyncio
 @dfly_args({"proactor_threads": 4})
 async def test_monitor_command(async_pool):
+    assert 1 == 2
```

Comment:
you do not want this

Reply:
this is for testing... I wanted to see that we dont get notifications